dashboard/pkg/epinio/pages/c/_cluster/dashboard.vue

```diff
@@ -94,6 +107,30 @@ export default Vue.extend<any, any, any, any>({
     }
   },
   methods: {
+    async calcAvailableResources() {
+      const nodeMetricsSchema = this.$store.getters[`epinio/schemaFor`](METRIC.NODE);
```

Comment:
Given the other comment about skipping the assignment for this in standalone world, we should gate first on !isSingleProduct and then fetch the schema

Reply:
Ah, you're right, I probably brake the `if` statement in the last change.